src/tools/python/valexp.md

```diff
@@ -0,0 +1,44 @@
+# Values and Expressions
```

Comment:
this file will not be shown on the summary, please modify SUMMARY.md with where you want this to be located.